pkg/cmd/server/api/register.go

```diff
@@ -0,0 +1,19 @@
+package api
+
+import (
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
+)
+
+var Scheme = runtime.NewScheme()
+
+func init() {
+	Scheme.AddKnownTypes("",
+		&OpenShiftMasterConfig{},
+		&KubernetesMasterConfig{},
+		&NodeConfig{},
+	)
+}
+
+func (*OpenShiftMasterConfig) IsAnAPIObject()  {}
+func (*KubernetesMasterConfig) IsAnAPIObject() {}
```

Comment:
do we ever plan to serialize this by itself? if not, kill it as a top-level type?